models/Search/Backend/Data/Listing/Dao.php

```diff
@@ -43,6 +43,7 @@ public function load()
             } elseif ($entryData['maintype'] == 'object') {
                 $element = DataObject::getById($entryData['id']);
             } else {
+                $element = null;
```

Comment:
just call `\Pimcore\Model\Element\Service::getElementById` and check only for `$entryData['maintype']`

Reply:
I used Service::getElementById now